src/gui/tray/activitylistmodel.cpp

```diff
@@ -394,6 +360,24 @@ void ActivityListModel::activitiesReceived(const QJsonDocument &json, int status
     }
 
     _activityLists.append(list);
+}
+
+void ActivityListModel::activitiesReceived(const QJsonDocument &json, int statusCode)
+{
+    const auto activities = json.object().value("ocs").toObject().value("data").toArray();
+
+    auto ast = _accountState;
```

Comment:
Why not just `if (_accountState)`? Also, why not `const auto ast`? Is this check even needed here with no immediate usage of `_accountState`?

Reply:
the `ast` name is probably a bad abbreviation
seeing the comment from @allexzander I was pretty sure we were doing some kind of abstract syntax trees
as a general rule we should avoid abbreviations as they are not easy to read later (or by somebody else)

Reply:
This was part of the code before and I hadn't changed it, but yes it is redundant so I have removed it now, thanks